### PR TITLE
Add computation utility class 

### PR DESCRIPTION
**Pull request by jamal-khey**
- Added a new dependency for powsybl-security-analysis-api version 6.4.1 in pom.xml.
- Created ComputationResultUtils class to handle violation location identification and formatting for network computations.

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**

**What kind of change does this PR introduce?**

**What is the current behavior?**

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?**
- [ ] Yes
- [ ] No

**If yes, please check if the following requirements are fulfilled**

- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration steps are described in the following section

**What changes might users need to make in their application due to this PR? (migration steps)**

**Other information**:

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -163,5 +170,11 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```
Comment (SlimaneAmar) [inline pom.xml:173]: Why this dependency ?

Comment (jamal-khey) [reply pom.xml]: i removed this dependency

File: src/test/java/com/powsybl/ws/commons/computation/ComputationUtilTest.java [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        VoltageLevel voltageLevel = network.getVoltageLevel("VLGEN");
+        Bus ngen = voltageLevel.getBusView().getBus("VLGEN_0");
+        when(busBreakerViolationLocation.getBusView(any())).thenReturn(() -> Stream.of(ngen));
+        assertEquals("VLGEN_0", ComputationResultUtils.getViolationLocationId(limitViolation, network));
+
+        Bus ngen2 = voltageLevel.getBusBreakerView().newBus()
+                .setId("NGEN2")
+                .add();
+
+        when(busBreakerViolationLocation.getBusView(any())).thenReturn(() -> Stream.of(ngen, ngen2));
```
Comment (SlimaneAmar) [inline src/test/java/com/powsybl/ws/commons/computation/ComputationUtilTest.java:114]: ngen2 is a configured bus and not a calculated bus !

Comment (jamal-khey) [reply src/test/java/com/powsybl/ws/commons/computation/ComputationUtilTest.java]: done

**Review by SlimaneAmar — APPROVED** [review]

File: src/test/java/com/powsybl/ws/commons/computation/ComputationUtilTest.java [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+import com.powsybl.ws.commons.computation.utils.ComputationResultUtils;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
```
Comment (EtienneLt) [inline src/test/java/com/powsybl/ws/commons/computation/ComputationUtilTest.java:18]: missing Author and headers

Comment (jamal-khey) [reply src/test/java/com/powsybl/ws/commons/computation/ComputationUtilTest.java:24]: fixed

File: src/main/java/com/powsybl/ws/commons/computation/utils/ComputationResultUtils.java [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+import com.powsybl.security.NodeBreakerViolationLocation;
+import com.powsybl.security.ViolationLocation;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static com.powsybl.iidm.network.IdentifiableType.BUSBAR_SECTION;
+import static com.powsybl.security.ViolationLocation.Type.NODE_BREAKER;
+
+public final class ComputationResultUtils {
```
[the remark this answers is not in the document]
Comment (jamal-khey) [reply src/main/java/com/powsybl/ws/commons/computation/utils/ComputationResultUtils.java:24]: fixed

**Pull request merged**